main.html

```diff
@@ -38,44 +38,252 @@ <h1 class="box h1">I’m Isaac Glad to see you!</h1>
+                                <p class="canopy">CANOPY</p>
+                            </div>
+                            <div class="div2-count1"></div>
+                            <div class="div3-back">
+                                <p class="backend">Back End Dev</p>
+                            </div>
+                            <div class="div4-count2"></div>
+                            <div class="div5-y15">
+                                <p class="y15">2015</p>
+                            </div>
+                        </div>
+                    </div>
+                    <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="lang">
+                        <li class="ht-1"><p class="ht">html</p></li>
+                        <li class="cs-1"><p class="cs">css</p></li>
+                        <li class="js-1"><p class="jscript">javaScript</p></li>
+                    </ul>
+                    <div class="butt">
+                        <button class="Acbutt"><p class="pbutt">See Project</p></button>
+                    </div>
+                </div>
+            </div>
+            <div class="con con4">
+                <div class="con-2 snapshot"><img src="img\multibg.png" alt="Project4"></div>
+                <div class="con-2 left-block4">
+                    <div class="l-block4-text">
+                        <p class="Text2">Multi-Post Stories</p>
+                        <div class="year">
+                            <div class="div1-can">
+                                <p class="canopy">CANOPY</p>
+                            </div>
+                            <div class="div2-count1"></div>
+                            <div class="div3-back">
+                                <p class="backend">Back End Dev</p>
+                            </div>
+                            <div class="div4-count2"></div>
+                            <div class="div5-y15">
+                                <p class="y15">2015</p>
+                            </div>
+                        </div>
+                    </div>
+                    <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="lang">
+                        <li class="ht-1"><p class="ht">html</p></li>
+                        <li class="cs-1"><p class="cs">css</p></li>
+                        <li class="js-1"><p class="jscript">javaScript</p></li>
+                    </ul>
+                    <div class="butt">
+                        <button class="Acbutt"><p class="pbutt">See Project</p></button>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+
+        <section class="about">
+            <div class="about-top">
+                <h1 class="about-header">About Myself</h1>
```

Comment:
- You are almost there, @gaks1 👍 
- [ ] How about adding a correct `background-color` to your document so that it matches the figma design?

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/212481290-b446405e-ab55-4af7-b0f7-a5a3869d0c72.png)|![image](https://user-images.githubusercontent.com/99715640/212481274-208dc22b-29f3-45a8-b0e7-72f23174a440.png)|

- [ ] I also think that you miss a `border-radius-right` on this very section so that you have that aesthetic curvy edge.
- [ ] On the `About Me` line, I think declaring a width will force the text to break into a new line as we have in the requirement.